test/to-esri.js

```diff
@@ -100,6 +100,32 @@ test('adding an object id', t => {
   t.end()
 })
 
+test('exclude objectid addition when not part of outfields', t => {
+  const options = {
+    toEsri: true,
+    outFields: ['string']
+  }
+  const fixture = _.cloneDeep(geojson)
+  const result = Winnow.query(fixture, options)
+  t.equal(Object.keys(result.features[0].attributes).length, 1)
+  t.equal(result.features[0].attributes.hasOwnProperty('string'), true)
+  t.end()
+})
+
+test('do not exclude objectid when returnIdsOnly = true', t => {
+  const options = {
+    toEsri: true,
+    outFields: ['string'],
+    returnIdsOnly: true
+  }
+  const fixture = _.cloneDeep(geojson)
+  const result = Winnow.query(fixture, options)
```

Comment:
hmm I'm not sure this is the correct response for returnIdsOnly

Reply:
I think this is right for Winnow.  Once it gets to FeatureServer, it get's transformed to:

```
{
  "objectIdFieldName" : "OBJECTID", 
  "objectIds" : [1]
}
```

Reply:
https://github.com/koopjs/FeatureServer/blob/master/src/query.js#L85-L94

Reply:
between the two libraries the logic is correct it but seems to me more of the logic should be pushed into winnow

Reply:
Seems like a good idea;  I can add changing winnow output of `returnIdsOnly` query as an issue.  Also - realizing that this PR is somewhat out of date;  this additional check for returnIdsOnly needs to be moved to the SQL function.